at-tanvo/D3-Exercise1.rb

```diff
@@ -0,0 +1,22 @@
+loop do
+  puts "Enter Number"
+  a = gets.to_i
+  if a>=9 && a<=10 
+    puts "A"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/c8f8a9c80a1053165f228ac8903bc3e0e0d677eb/at-tanvo/D3-Exercise1.rb#L5"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>